modules/mobile/config/initializers/statsd.rb

```diff
@@ -8,3 +8,10 @@
 Mobile::ApplicationController.statsd_count_success :authenticate, 'mobile.authentication'
 StatsD.increment('mobile.authentication.success', 0)
 StatsD.increment('mobile.authentication.failure', 0)
+
+Mobile::V0::Appointments::Service.extend StatsD::Instrument
```

Comment:
@kreek lets think about what other mobile metrics we may want to capture.

Reply:
👍 my plan is to get the initial set of endpoints in to unblock the FE then circle back have a longer think about which metrics we want to capture.